src/test/run-make-fulldeps/issue-72550/linux.stderr

```diff
@@ -0,0 +1,17 @@
+thread 'main' panicked at 'explicit panic', test.rs:2:5
+stack backtrace:
+   0: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
+   1: core::fmt::write
+   2: std::io::Write::write_fmt
+   3: std::panicking::default_hook::{{closure}}
+   4: std::panicking::default_hook
+   5: std::panicking::rust_panic_with_hook
+   6: std::panicking::begin_panic
+   7: test::main
+   8: std::rt::lang_start::{{closure}}
+   9: std::rt::lang_start_internal
+  10: std::rt::lang_start
+  11: main
```

Comment:
interestingly, this also does not have line numbers, but when I run the test in a docker image (rust latest, `rustc 1.43.1 (8d69840ab 2020-05-04)`) it *does* have line numbers.

Note that I'm not setting the same environment variables as compiletest beause I have other problems with my rustc build at the moment, so it might be that compiletest is explicitly passing `RUSTC=$(something that causes it to not emit debug symbols)` or something. I will look into this next.